docs/conf.py

```diff
@@ -55,10 +56,22 @@
 # |version| and |release|, also used in various other places throughout the
 # built documents.
 #
+
+__version__ = '0.0.0'
+
+cwd = dirname(realpath(__file__))
+root = abspath(join(cwd, '..'))
+
+with open(join(root, '%s/_version.py' % project)) as version_file:
+    for line in version_file:
+        # This will populate the __version__ and __version_info__ variables
+        if line.startswith('__'):
+            exec(line)
+
 # The short X.Y version.
-version = '0.1.0'
+version = __version__
 # The full version, including alpha/beta/rc tags.
-release = '0.1.0'
+release = __version__
```

Comment:
It's an option Sphinx provides to have the two separate. It's not something we're using in any of our existing versioning schema